lab-noah/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "lab-noah",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "build": "./node_modules/webpack/bin/webpack.js",
+    "watch": "./node_modules/webpack-dev-server/bin/webpack-dev-server.js --inline --hot"
+  },
```

Comment:
the assignment required a lint script such as in addition "lint": "./node_modules/eslint/bin/eslint.js .",  plug a lint script in